utils/functions

```diff
@@ -33,6 +33,13 @@
                  34sns-repair-1n-1f
                  35m0singlenode
                  36spare-reservation
+# 37protocol ST checks checksum of the BE structures during
+# developmental phase they are bound to fail so code
+# changes are required but ST has to be in disabled
+# state for now when the release is in GA phase we will
+# enable the ST again to ensure there will be no changes
+# in BE structures.
```

Comment:
```suggestion
# this ST confirms that no BE structures are changed
# in the new version of the code since that would 
# create a mismatch of BE structures and thus cause 
# corruption in case of upgrades. During development 
# phase the BE structures are assumed to be modified
# and to avoid this ST from failing it is better to disable
# the ST until a GA release is done. After GA release this
# ST needs to be enabled to ensure that BE structure 
# changes are allowed after a careful consideration.
```

Reply:
This is no longer needed as I updated skip list as per upendra's comment

Reply:
Make sure this test is not run from within Jenkins which will stop us from committing the code.